docs/en/database_engines/mysql.md

```diff
@@ -0,0 +1,119 @@
+#MySQL
+
+Allows to exchange data with the remote MySQL server.
+
+You can connect to some database on the MySQL server and perform `INSERT` and `SELECT` queries with tables.
```

Comment:
I'd merge this with previous sentence, for example:

Allows to connect to some  database on remote MySQL server and perform `INSERT` and `SELECT` queries with  tables to exchange data between ClickHouse and MySQL.